src/routes/+layout.js

```
@@ -1,8 +1,11 @@
-import { fetchProjects } from "$lib/projects.svelte.js";
+import { fetchProjects, loadProjectsClicked } from "$lib/projects.svelte.js";
 
 export const load = async () => {
 	try {
 		await fetchProjects();
+		if (typeof window !== undefined && localStorage !== undefined) {
```

Biome (2.1.2)

[error] Invalid typeof comparison.

Compare with one of the following string literals:


(lint/correctness/useValidTypeof)